src/OpenDiffix.Core/Utils.fs

```diff
@@ -86,3 +86,17 @@ module Hash =
 module Math =
   let roundAwayFromZero (x: float) : float =
     System.Math.Round(x, MidpointRounding.AwayFromZero)
+
+let TIMESTAMP_REFERENCE = DateTime(1800, 1, 1, 0, 0, 0, DateTimeKind.Utc)
+
+// Encodes `s` as a double-precision floating point number, obtained by hashing.
+// Uses only a subset of representable floating point to avoid large/small numbers.
+// For columns with many unique values collisions are possible, but collisions numerous enough
+// to cross a reasonable LCF threshold rather unlikely.
+let hashStringToFloat (s: String) =
```

Comment:
We recently changed it to map to a value `0..1`, but either will be replaced soon.